scrap/db.py

```
@@ -0,0 +1,103 @@
+def get_book_by_title_author(title: str, author: str):
+    sql = """
+        SELECT id
+        FROM book
+        WHERE title = %s
+          AND author = %s
+        LIMIT 1
+    """
+    return fetch_one(sql, (title, author))
+
+
+def insert_book(
```

Ruff (0.11.9)

Missing return type annotation for public function `insert_book`

(ANN201)